### DISCUSSION TITLE
Extra版本训练出的模型有概率会出现声音模糊不清

### DISCUSSION THREAD

**Discussion by oszc** [fishaudio/Bert-VITS2#291] [Q&A]
我测试了以下：G_8000.pth文件吐词正常
G_57000.pth模型文件则出现吐词不清问题

**Comment by oszc** [comment]
吐词不清是指文字和语音完全对不上，发音完全不准确

**Comment by glcolor** [comment]
我也遇到了这个问题，请问您解决了吗？

**Reply by oszc** [reply]
我的解决办法就是少训练一些步数，比如只训练到G_10000就结束，挑选前面能用的声音。最后一个能用的模型效果未必是最好的。反而前面的一些模型效果比后面训练的模型更好一些。

**Reply by glcolor** [reply]
明白了，感谢指点~

**Comment by qicrafts** [comment]
楼主特化版本用的是哪个底模，是【Bert-VITS2_中文特化底模】还是【Bert-VITS2_2.3底模】？

**Reply by oszc** [reply]
Bert-VITS2_中文特化底模，这个底膜效果不错，只是在我这不能训练很多轮

**Reply by qicrafts** [reply]
好的,多谢楼主~ 另外请教楼主有没有出现合成的声音，上气不接下气，好像急着要赶紧把句子读完的现象？

**Reply by starmoon-1134** [reply]
我也遇到了，训练到大约50k step的时候，g/dur和g/kl突然上涨，然后就炸炉了
<img width="707" alt="20240130-bert-vits2-loss" src="https://github.com/fishaudio/Bert-VITS2/assets/16525284/d6b2c9b9-0641-481c-8c18-d18c26dee074">

合成的声音很奇怪，都不是正常的声音
<img width="1087" alt="image" src="https://github.com/fishaudio/Bert-VITS2/assets/16525284/c9858d48-eeca-4081-92ac-fd8da9dcce4c">

**Comment by Stardust-minus** [comment]
考虑降学习率，加大deacy，加大bs，打开梯度裁剪
